dcs/terrain/germanycoldwar/germanycoldwar.py

```diff
@@ -24,15 +24,15 @@ class GermanyColdWar(Terrain):
     assert len(temperature) == 12
 
     def __init__(self):
-        bounds = mapping.Rectangle(-425000.0, -600000.0, -1100000.0, 260000.0, self)
+        bounds = mapping.Rectangle(260000.0, -1100000.0, -600000.0, -425000.0, self)
```

Comment:
This is actually what fixed it O believe, I can't believe I didn't spot this earlier because it's quite obvious that our bounds were simply stated in the wrong order 😅